pandas/io/json/_normalize.py

```diff
@@ -531,7 +531,11 @@ def _recursive_extract(data, path, seen_meta, level=0):
             raise ValueError(
                 f"Conflicting metadata name {k}, need distinguishing prefix "
             )
-        result[k] = np.array(v, dtype=object).repeat(lengths)
+        out = []
+        for item, repeat in zip(v, lengths):
```

Comment:
Can this be done *if* we know the values are nested, otherwise, just use the `np.array` constructor?

Reply:
done